common/ledger/blockledger/ledger.go

```diff
@@ -17,6 +17,9 @@ type Factory interface {
 	// or creates it if it does not
 	GetOrCreate(channelID string) (ReadWriter, error)
 
+	// Remove removes an existing ledger
+	Remove(channelID string) error
```

Comment:
Add some documentation that this operation is blocking and may take some time, as it cleans the index and then removes the folder.

Reply:
We may need to run some benchmarks that will estimate how long this may take for different ledger sizes. This will allow us to decide whether the upper layer will spawn a go-routine to do this or it is enough to do it on the request go-routine.

Reply:
Updated the comment. The other portion seems worthy of a follow-up task once we're able to actually perform the RemoveChannel operation.